packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -200,12 +195,34 @@ class VifAllowedIps extends BaseComponent {
       find(ipPool.networks, ipPoolNetwork => ipPoolNetwork === vifNetworkId)
   )
 
-  _toggleNewIp = () =>
-    this.setState({ showNewIpForm: !this.state.showNewIpForm })
+  _hideIpInputs = () =>
+    this.setState({
+      showIpSelector: false,
+      showIpInput: false,
+    })
+
+  _onKeyDown = async event => {
+    const { keyCode, target } = event
+
+    if (keyCode === 13) {
```

Comment:
IMHO, using a [`KeyboardEvent.key`](https://developer.mozilla.org/fr/docs/Web/API/KeyboardEvent/key/Key_Values) will be more understandable than the key code.